x-pack/platform/plugins/shared/inference/server/config.ts

```diff
@@ -9,6 +9,13 @@ import { schema, type TypeOf } from '@kbn/config-schema';
 
 export const configSchema = schema.object({
   enabled: schema.boolean({ defaultValue: true }),
+  regexWorker: schema.object({
```

Comment:
Is this is supposed to be user facing setting then:
1. We might want a better naming,`regexWorker` sounds pretty terrible to me?
2. It needs to be added to our public documentation
3.(IIRC)  it needs to be added to the docker file where we have to hardcode all the settings, if that monstrosity is still a thing

Reply:
@dgieselaar I hadn't seen your comment, previously.   Sure.